addOns/ascanrules/src/test/java/org/zaproxy/zap/extension/ascanrules/TestCrossSiteScriptV2UnitTest.java

```diff
@@ -760,13 +968,13 @@ protected Response serve(IHTTPSession session) {
 
         this.nano.addHandler(handler);
 
+        // When
         HttpMessage msg = this.getHttpMessage(test + "?name=test");
         msg.getRequestHeader().setMethod(HttpRequestHeader.PUT);
 
         this.rule.setConfig(new ZapXmlConfiguration());
         this.rule.setAlertThreshold(AlertThreshold.LOW);
         this.rule.init(msg, this.parent);
-        // When
```

Comment:
same

Reply:
i have reset the changed lines.
thanks,
karan